tests/checkers/inference/NninfTest.java

```diff
@@ -0,0 +1,31 @@
+package checkers.inference;
+
+import checkers.inference.solver.MaxSat2TypeSolver;
+import checkers.inference.test.CFInferenceTest;
+import org.checkerframework.framework.test.TestUtilities;
+import org.checkerframework.javacutil.Pair;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class NninfTest extends CFInferenceTest {
+
+    public NninfTest(File testFile) {
+        super(testFile, nninf.NninfChecker.class, "nninf",
+                "-Anomsgtext", "-d", "tests/build/outputdir");
+    }
+
+    @Override
+    public Pair<String, List<String>> getSolverNameAndOptions() {
+        return Pair.<String, List<String>>of(MaxSat2TypeSolver.class.getCanonicalName(), new ArrayList<String>());
```

Comment:
It seems like the type arguments can be inferred. Can we try to remove them?

Reply:
Yes, we should remove them. I'm not sure why we have these type arguments in other subclasses.